pmm/aws-staging-start-pmm2.groovy

```diff
@@ -0,0 +1,383 @@
+                        fi
+
+                        ssh -i "${KEY_PATH}" -o ConnectTimeout=1 -o StrictHostKeyChecking=no ${USER}@\$(cat IP) "
+                            set -o errexit
+                            set -o xtrace
+
+                            sudo yum -y update --security
+                            sudo yum -y install https://repo.percona.com/yum/percona-release-0.1-7.noarch.rpm
+                            sudo rpm --import /etc/pki/rpm-gpg/PERCONA-PACKAGING-KEY
+                            sudo yum -y install svn docker sysbench mysql57-server
+                            sudo service mysqld start
+                            sudo yum -y install bats --enablerepo=epel
+                            sudo usermod -aG docker ec2-user
+                            sudo service docker start
+
+                            sudo mkdir -p /srv/pmm-qa || :
+                            pushd /srv/pmm-qa
+                                sudo svn export https://github.com/percona/pmm-qa.git/trunk/pmm-tests
+                                sudo svn export https://github.com/Percona-QA/percona-qa.git/trunk/get_download_link.sh
+                                sudo chmod 755 get_download_link.sh
+                            popd
+                        "
+                    """
+                }
+                script {
+                    env.IP      = sh(returnStdout: true, script: "cat IP").trim()
+                    env.VM_NAME = sh(returnStdout: true, script: "cat VM_NAME").trim()
+                }
+                archiveArtifacts 'IP'
+            }
+        }
+
+        stage('Run Docker') {
+            steps {
+                withCredentials([sshUserPrivateKey(credentialsId: 'aws-jenkins', keyFileVariable: 'KEY_PATH', passphraseVariable: '', usernameVariable: 'USER')]) {
+                    sh """
+                        export IP=\$(cat IP)
+                        export VM_NAME=\$(cat VM_NAME)
+
+                        export CLIENT_VERSION=${CLIENT_VERSION}
+                        if [[ \$CLIENT_VERSION = latest ]]; then
+                            CLIENT_VERSION=\$(
+                                curl -s https://www.percona.com/downloads/pmm/ \
+                                    | egrep -o 'pmm/[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}' \
+                                    | sed -e 's/pmm\\///' \
+                                    | sort -u -V \
+                                    | tail -1
+                            )
+                        fi
+
+                        ssh -i "${KEY_PATH}" -o ConnectTimeout=1 -o StrictHostKeyChecking=no ${USER}@\$(cat IP) "
+                            set -o errexit
+                            set -o xtrace
+
+                            docker create \
+                                -v /opt/prometheus/data \
```

Comment:
as I know for PMM2 we don't need to specify paths for grafana/consult etc, 
docker create   -v /srv    --name ${VM_NAME}-data      ${DOCKER_VERSION} /bin/true